README.md

```diff
@@ -88,6 +88,27 @@ var_dump($token->validate($data)); // true, because current time is between "nbf
 $data->setCurrentTime(time() + 4000); // changing the validation time to future
 
 var_dump($token->validate($data)); // false, because token is expired since current time is greater than exp
+
+// We can also use leeway to deal with clock skew.
```

Comment:
I think most readers of the `README.md` will unlikely know what `leeway` or `clock skew` mean in this context. Also, `iat`, `nbf` and `exp`, while in the spec, need some expansion here.

Is this maybe for a separate documentation page? (`README.md` should be short and to the point)

Reply:
> Is this maybe for a separate documentation page? (README.md should be short and to the point)

I agree with you @Ocramius but let's do that as a separate PR, we can keep it on the `README.md` for now and modify it later.